src/components/vice/admin/index.js

```diff
@@ -191,12 +224,146 @@ const VICEAdminSkeleton = () => {
     );
 };
 
+const VICEAdminTabs = ({ data = {} }) => {
+    const classes = useStyles();
+
+    const [value, setValue] = useState("0");
+
+    const tabID = (name) => id(ids.ROOT, "admin", "tabs", name);
+    const tabPanelID = (name) => id(ids.ROOT, "admin", "tab-panels", name);
+    const ariaControls = (name) => `vice-admin-tabs-${name}`;
+
+    const [mutantExit] = useMutation(exit, {
+        onSuccess: () => queryCache.refetchQueries(getDataQueryName),
+    });
+    const [mutantSaveAndExit] = useMutation(saveAndExit, {
+        onSuccess: () => queryCache.refetchQueries(getDataQueryName),
+    });
+    const [mutantExtendTimeLimit] = useMutation(extendTimeLimit, {
+        onSuccess: () => queryCache.refetchQueries(getDataQueryName),
+    });
+    const [mutantUploadOutputs] = useMutation(saveOutputFiles, {
+        onSuccess: () => queryCache.refetchQueries(getDataQueryName),
+    });
+    const [mutantDownloadInputs] = useMutation(downloadInputFiles, {
+        onSuccess: () => queryCache.refetchQueries(getDataQueryName),
+    });
+
+    const [analysisRows, setAnalysisRows] = useState([]);
+
+    useEffect(() => {
+        setAnalysisRows(getAnalyses(data));
+    }, [data]);
+
+    const orderOfTabs = [
+        "analyses",
+        "deployments",
+        "services",
+        "pods",
+        "configMaps",
+        "ingresses",
+    ];
+
+    return (
+        <TabContext value={value}>
+            <AppBar
+                position="static"
+                color="primary"
+                classes={{ colorPrimary: classes.tabAppBarColorPrimary }}
+            >
+                <TabList
+                    onChange={(_, newValue) => setValue(newValue)}
+                    variant="scrollable"
+                    scrollButtons="auto"
+                    indicatorColor="secondary"
+                >
+                    {orderOfTabs.map((tabName, index) => (
+                        <Tab
+                            label={msg(tabName)}
+                            id={tabID(tabName)}
+                            key={tabID(tabName)}
+                            value={`${index}`}
+                            aria-controls={ariaControls(tabName)}
```

Comment:
Minor - everybody gets this part confused - but `aria-controls` is supposed to establish a relationship by ID, it's not a label. So the tab's value in `aria-controls` should match the ID of the tab panel it refers to.  It's not 100% clear in the MUI examples, but it is there.  I think in your case, this should be `tabPanelID(tabName)`

Reply:
Thanks for pointing this out, should be fixed now.